package.json

```diff
@@ -63,7 +63,7 @@
     "@types/jest": "24.0.18",
     "@types/moment-timezone": "0.5.12",
     "@types/node": "12.7.11",
-    "@zerollup/ts-transform-paths": "1.7.3",
+    "@zerollup/ts-transform-paths": "~1.7.3",
```

Comment:
This had to be cherry-picked also or else the webapp build would not be able to pass - functionally it changes nothing